src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1086,7 +1088,7 @@ format layout_optimizer::get_expected_format(quantize_node const& node) {
         return all_users_gemm;
     };
 
-    auto use_onednn_impls = _optimization_attributes.use_onednn_impls;
+    auto use_onednn_impls = contains_onednn_impls_optimization_attribute(&node);
```

Comment:
This will always return false, won't it? Probably it should be `has_all_enabled_onednn_impls_optimization_attribute()` to be compatible with previous behavior, or it should check the type of the users node directly

Reply:
done